deps/node-inspect/lib/internal/inspect_repl.js

```diff
@@ -84,7 +84,7 @@ function extractFunctionName(description) {
   return fnNameMatch ? `: ${fnNameMatch[1]}` : '';
 }
 
-const NATIVES = process.binding('natives');
+const NATIVES = internalBinding('natives');
```

Comment:
Can you undo changes to `deps/`?

Also … @jkrems do you know what we could do to avoid this?